### DISCUSSION TITLE
Missing columns in csv export. Cannot export casting boolean fields.

### DISCUSSION THREAD

**Discussion by cable8mm** [laravel/nova-issues#5201] [Q&A] [answered]
- Laravel Version: 9.45.1
- Nova Version: v4.19.5
- PHP Version: 8.1.13
- Database Driver & Version: mysql & 8.0.31
- Operating System and Version: macOS Ventura 13.0.1
- Browser type and version: Chrome 108.0.5359.124 (Official Build) (x86_64)
- Reproduction Repository: https://github.com/###/###

### Description:

`has_caption` and `is_active` are tinyint type.

Model:

```php
    protected $casts = [
        'has_caption' => 'boolean',
        'is_active' => 'boolean',
    ];

```

Missing two columns in csv export.

### Detailed steps to reproduce the issue on a fresh Nova installation:

1. Create a table & model include tinyint type columns.
2. Cast boolean.
3. Create resource.
4. Add `ExportAsCsv::make()` action.
5. Export as csv.
6. Missing the field.

**Comment by crynobone** [comment, marked as answer]
You can use `withFormat()` : https://nova.laravel.com/docs/4.0/resources/#csv-export
